openmdao/core/tests/test_group.py

```diff
@@ -2198,14 +2199,14 @@ def test_conflicting_units(self):
         with self.assertRaises(Exception) as cm:
             p.setup()
 
-        self.assertEqual(cm.exception.args[0], "Groups 'par.G4' and 'par.G5' added the input 'x' with conflicting 'units'.")
+        self.assertEqual(cm.exception.args[0], "Group (<model>): The subsystems G1.G2 and par.G4 called set_input_defaults for promoted input 'x' with conflicting values for 'units'. Call <group>.set_input_defaults('x', units=?), where <group> is the Group named '' to remove the ambiguity.")
```

Comment:
I wonder if we should have special handling for the top level group...  
It seems like this message would make more sense to the user if it referred to "the 'model'" instead of "the group named ''", since that's how the user would reference it.

Reply:
changed "Group named ''" to "model"